client/my-sites/site-logs/components/site-logs-table/site-logs-expanded-content.tsx

```diff
@@ -0,0 +1,23 @@
+import { SiteLogsData } from 'calypso/data/hosting/use-site-logs-query';
+import './style.scss';
+
+interface Props {
+	log: SiteLogsData[ 'logs' ][ 0 ];
+}
+
+export default function SiteLogsExpandedContent( { log }: Props ) {
+	return (
+		<div className="site-logs-table__expanded-content">
```

Comment:
We also really shouldn't be using `<tr>` elements when they're not children of a `<table>` or `<tbody>`. You could replace the `.site-logs-table__expanded-content` element with a `<table>`. Or the other option could be to use CSS grid to lay out the expanded content. e.g.

```tsx
		<div className="site-logs-table__expanded-content" style={{
			display: 'grid',
			gridTemplateColumns: 'auto 1fr',
			gap: '0.6em 1em'
		}}>
			{ Object.keys( log ).map( ( key ) => (
				<Fragment key={ key }>
					<div>
						<strong>{ key }</strong>
					</div>
					<div>
						<div className="site-logs-table__expanded-content-info">{ log[ key ] || '-' }</div>
					</div>
				</Fragment>
			) ) }
		</div>
```

(inline styling for demonstration purposes only 😄)